static/app/components/sidebar/index.tsx

```diff
@@ -91,6 +91,14 @@ function Sidebar({location, organization}: Props) {
     return () => bcl.remove('body-sidebar');
   }, [bcl]);
 
+  useEffect(() => {
+    Object.values(SidebarPanelKey).forEach(sidebarValue => {
+      if (location?.hash === `#sidebar-${sidebarValue}`) {
```

Comment:
```suggestion
    Object.values(SidebarPanelKey).forEach(key => {
      if (location?.hash === `#sidebar-${key}`) {
```

🤷 what is 'value' lol